lib/pagseguro/items.rb

```diff
@@ -24,7 +30,9 @@ def <<(item)
     # Verify if the item is already included to item list.
     # Returns boolean.
     def include?(item)
-      @store.find {|stored_item| stored_item.id == ensure_type(Item, item).id }
+      @store.detect do |stored_item|
+        stored_item.id == item.id && stored_item.description == item.description
```

Comment:
cool.
if it's possibe to add items with the id, why does it need the comparison? maybe it's not necessary. 
what do you guys think about it?